server/metrics/metrics.go

```diff
@@ -57,23 +101,23 @@ func New(config *libconfig.MetricsConfig) *Metrics {
 			prometheus.CounterOpts{
 				Subsystem: collectors.Subsystem,
 				Name:      collectors.Operations.String(),
-				Help:      "Cumulative number of CRI-O operations by operation type.",
+				Help:      "[DEPRECATED] Cumulative number of CRI-O operations by operation type.",
```

Comment:
Let's add in which favor it has been deprecated.

Reply:
b1a1bfa.